include/seqan3/alphabet/alphabet_base.hpp

```diff
@@ -15,7 +15,7 @@
 #include <cassert>
 
 #include <seqan3/alphabet/concept.hpp>
-#include <seqan3/core/detail/int_types.hpp>
+#include <seqan3/utility/detail/int_types.hpp>
```

Comment:
do we sort these? or do we do that afterwards (this makes merge conflict less likely)?

Reply:
I would sort them right away.